app/models/endpoint.rb

```diff
@@ -41,6 +41,18 @@ def ssl_cert_store
     nil # use system defaults
   end
 
+  # Split concatenated PEM certs to a list
+  def certificate_authorities
+    return [] if certificate_authority.blank?
+    certificate_authority.split(/(?=-----BEGIN)/).reject(&:blank?)
```

Comment:
I think that the safest approach is to eliminate completely any text that isn't between the begin and end marks.